ts/features/fims/history/screens/HistoryScreen.tsx

```diff
@@ -64,10 +64,10 @@ export const FimsHistoryScreen = () => {
       ? Date.now() - lastErrorToastDate.current >= 500
```

Comment:
same here, why not rename `consents` to `accesses`, since it is now the "chosen" naming ?